### PR TITLE
Update release 1.13 in README file.

### PR DESCRIPTION
**Pull request by vanbasten23**
Merge after we publish and validate all images.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,22 +63,22 @@ If a single TPU VM does not suit your requirment, you can consider using TPU Pod
 ## <a name="Resource"></a> Available images and wheels
 The following pre-built docker images are available to run on Cloud TPU Nodes (see [docker images](#DockerImage) for instructions):
 
-    * `gcr.io/tpu-pytorch/xla:r1.12_3.7`: The current stable version.
-    * `gcr.io/tpu-pytorch/xla:r1.11_3.7`: The 1.11 release version.
+    * `gcr.io/tpu-pytorch/xla:r1.13_3.7`: The current stable version.
+    * `gcr.io/tpu-pytorch/xla:r1.12_3.7`: The 1.12 release version.
```
Comment (JackCaoG) [inline README.md:67]: let's keep the 1.11 one here too

Comment (vanbasten23) [reply README.md:67]: done

**Comment by JackCaoG** [comment]
we should update and merge this pr, all wheels should be out.

**Comment by vanbasten23** [comment]
Also, I've checked the docker image is ready:
```
xiowei@t1v-n-2a2b95ef-w-0:~$ sudo docker pull gcr.io/tpu-pytorch/xla:r1.13_3.8_tpuvm
r1.13_3.8_tpuvm: Pulling from tpu-pytorch/xla
fd31ec14dccd: Already exists
67ecdff6b177: Pull complete
52dfe9565a69: Pull complete
```

**Pull request merged**